### PR TITLE
ignite-2.8.1-release-notes

### PR DESCRIPTION
**Pull request by nizhikov**
Ignite 2.8.1 Release Notes.

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-12407: Add Cluster API support to Java thin client`
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: RELEASE_NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,88 @@
...
+* Fixed Can't set serialized enum to a BinaryObject's field
+* Fixed Prevent partitions owned by other nodes switch their state to MOVING due to counter difference on node join.
+* Fixed Some of rentingFutures in GridDhtPartitionTopologyImpl may accumulate a huge number of eviction callbacks
+* Fixed It is needed to set used cache for Spring Data dynamically
+* Fixed Inconsistency between cache data and indexes when cache operation is interrupted
+* Fixed MetricRegistryMBean is not thread safe
+* Fixed Update of partition's states can stuck when rebalance completed during exchange
+* Fixed Non-comparable keys for eviction policy cause failure handle and node shutdown
+* Fixed Historical (WAL) rebalance can start on a cleared partition if some baseline node leaves the cluster and then joins back.
+* Fixed ClassCastException on thinClient in Apache Ignite
```
Comment (Mmuzaf) [inline RELEASE_NOTES.txt:64]: thinClient -> thin client

File: RELEASE_NOTES.txt [hunk 2]
```diff
@@ -1,6 +1,88 @@
...
 
+
+Apache Ignite In-Memory Database and Caching Platform 2.8.1
+-----------------------------------------------------------
+
+Security:
+* Fixed vulnerability of using H2 admin rights for user SQL connections
+
+Ignite Core:
+* Added transaction operations metrics
```
Comment (Mmuzaf) [inline RELEASE_NOTES.txt:12]: I'd suggest sorting these lines in alphabetical order (e.g. in the IntelliJ IDEA `Edit -> Sort lines`).

File: RELEASE_NOTES.txt (hunk 1 above)
Comment (nizhikov) [reply RELEASE_NOTES.txt:64]: fixed.

File: RELEASE_NOTES.txt (hunk 2 above)
Comment (nizhikov) [reply RELEASE_NOTES.txt]: fixed.

File: RELEASE_NOTES.txt [hunk 3]
```diff
@@ -1,6 +1,88 @@
 Apache Ignite Release Notes
 ===========================
 
+
+Apache Ignite In-Memory Database and Caching Platform 2.8.1
```
Comment (dmagda) [inline RELEASE_NOTES.txt:5]: Let's rename to "Apache Ignite In-Memory Computing Platform" as per the website. The rest looks good, thanks!

Comment (nizhikov) [reply RELEASE_NOTES.txt:5]: fixed.

**Pull request merged**